modules/ingester/ingester.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/grafana/tempo/modules/overrides"
 	"github.com/grafana/tempo/modules/storage"
 	"github.com/grafana/tempo/pkg/flushqueues"
+	_ "github.com/grafana/tempo/pkg/gogocodec" // force gogo codec registration
```

Comment:
Just making a note here that the querier also calls the ingester, but not with any `customtype`.